.github/workflows/chromatic.yaml

```diff
@@ -0,0 +1,44 @@
+name: Chromatic
+
+on:
+  push:
+    branches-ignore:
+      - master-mobile
+      - master-frontend
+      - master-backend
+      - staging-mobile
+      - staging-frontend
+      - staging-backend
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    environment: checks
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: "20.13.1"
+
+      - name: Install pnpm
+        uses: pnpm/action-setup@v4
+
+      - name: Install Dependencies
+        run: |
+          pnpm add -g turbo
+          pnpm install --frozen-lockfile
+
+      - name: Run Chromatic
+        uses: chromaui/action@latest
+        with:
+          # ⚠️ Make sure to configure a `CHROMATIC_PROJECT_TOKEN` repository secret
+          projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
```

Comment:
The CHROMATIC_PROJECT_TOKEN is only used by the CI I assume ?

Reply:
No you need a .env file in apps/storybook too.  😉